homeassistant/requirements.py

```diff
@@ -50,6 +57,29 @@ def __init__(self, domain: str, requirements: List) -> None:
     if hass.config.skip_pip:
         return integration
 
+    cache = hass.data.get(DATA_INTEGRATIONS_WITH_REQS)
+    if cache is None:
+        if not _async_mount_config_dir(hass):
+            raise IntegrationNotFound(domain)
```

Comment:
This is not necessary. This is only for resolving integrations, which this function outsources to `async_get_integration`


```suggestion
```

Reply:
Yeah, I thought about leaving that out with a comment, but I wasn't sure it it would be safe to assume it would always be handled elsewhere.